src/xenia/kernel/user_module.h

```diff
@@ -97,9 +98,11 @@ class UserModule : public XModule {
 
  private:
   X_STATUS LoadXexContinue();
+  void CalculateHash();
 
   std::string name_;
   std::string path_;
+  uint64_t hash_ = -1;
```

Comment:
`UINT64_MAX` or `~UINT64_C(0)` as this is unsigned. Though neither this nor even 0 is an invalid hash value — if such indication is needed, `std::optional` should be preferred.